lib/runtime/test_helpers.go

```diff
@@ -80,30 +80,24 @@ func GetAbsolutePath(targetDir string) string {
 }
 
 // GetRuntimeBlob checks if the test wasm @testRuntimeFilePath exists and if not, it fetches it from @testRuntimeURL
-func GetRuntimeBlob(testRuntimeFilePath, testRuntimeURL string) (n int64, err error) {
+func GetRuntimeBlob(testRuntimeFilePath, testRuntimeURL string) error {
 	if utils.PathExists(testRuntimeFilePath) {
-		return 0, nil
+		return nil
 	}
 
-	out, err := os.Create(testRuntimeFilePath)
+	/* #nosec */
+	resp, err := http.DefaultClient.Get(testRuntimeURL)
```

Comment:
Careful the default client has no timeout AFAIK, so you might want to create your own client with a timeout (either here or pass it as argument). Using the default client is not ideal either, since it might conflict with other tests.

Reply:
Changed to `http.Get`